packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -295,18 +425,13 @@ - (void)imagePickerController:(UIImagePickerController *)picker
     if (image == nil) {
       image = [info objectForKey:UIImagePickerControllerOriginalImage];
     }
+    __weak typeof(self) weakSelf = self;
 
     NSNumber *maxWidth = [_arguments objectForKey:@"maxWidth"];
     NSNumber *maxHeight = [_arguments objectForKey:@"maxHeight"];
-    NSNumber *imageQuality = [_arguments objectForKey:@"imageQuality"];
+    NSNumber *imageQuality = [weakSelf.arguments objectForKey:@"imageQuality"];
```

Comment:
nits: same, doesn't need to be weak here and below

Reply:
Done!